src/main/java/websocket/models/requests/UserRegisterRequest.java

```diff
@@ -0,0 +1,45 @@
+package websocket.models.requests;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import websocket.models.IRequestData;
+import websocket.models.Request;
+//import
+
+public class UserRegisterRequest implements IRequestData {
+
+    @JsonProperty("Username")
+	protected String username;
+
+    @JsonProperty("FirstName")
+	protected String firstName;
+
+    @JsonProperty("LastName")
+	protected String lastName;
+
+    @JsonProperty("Email")
+	protected String email;
+
+    @JsonProperty("Password")
+	protected String password;
```

Comment:
the indentation on these fields and the method below needs to be aligned like it is in the other files